src/load_dotenv.rs

```diff
@@ -1,43 +1,57 @@
 use crate::common::*;
 
-pub(crate) fn load_dotenv(
+fn load_from_file(
   config: &Config,
   settings: &Settings,
-  working_directory: &Path,
+  path: &Path,
 ) -> RunResult<'static, BTreeMap<String, String>> {
   // `dotenv::from_path_iter` should eventually be un-deprecated, see:
   // https://github.com/dotenv-rs/dotenv/issues/13
   #![allow(deprecated)]
 
+  if settings.dotenv_load.is_none()
```

Comment:
This warning concerns implicit loading of `.env` files, as some future version of `just` will switch from loading `.env` files automatically to only on request.

If the user passes either `--dotenv-path` or `--dotenv-filename`, then this warning shouldn't be printed, since they explicitly requested that an environment file be loaded.